alembic/versions/3741581c7fc4_initial.py

```diff
@@ -149,6 +149,7 @@ def upgrade():
                     sa.Column('safety_measures_use', sa.Integer(), nullable=True),
                     sa.Column('late_deceased', sa.Integer(), nullable=True),
                     sa.ForeignKeyConstraint(['accident_id', 'provider_code'], ['markers.id', 'markers.provider_code'],
+                                            name='involved_accident_id_fkey',
```

Comment:
Hi, I did not see this `name=` parameter anywhere else in our code. Do you understand why it is needed here?

Reply:
Yes I do, if I don't name it, in a following migration we're deleting a foreign key by name, and it fails because the automatic naming might have changed (it has become nvolved_accident_id_provider_code_fkey in this case)